### ISSUE TITLE
gradle

### ISSUE THREAD

**Issue by noctric** [noctric/FileBin-mobile#3]
update/fix gradle config

**Comment by sebastianrakel** [comment]
Is there any progress? It's blocks development on this project :D

**Comment by tw-michael** [comment]
Not really, have been debugging yesterday but did not come to any useful conclusion. It basically seems to have broken with some upgrade (either gradle or android studio itself).
Feel free to try and fix this! Unfortunately I'm currently busy with a lot of work, even on the weekends :(

**Comment by noctric** [comment]
Not really, have been debugging yesterday but did not come to any useful conclusion. It basically seems to have broken with some upgrade (either gradle or android studio itself).
Feel free to try and fix this! Unfortunately I'm currently busy with a lot of work, even on the weekends :(

**Comment by sebastianrakel** [comment]
Ah its running again :+1:
